source/slang/slang-emit-spirv.cpp

```diff
@@ -5553,6 +5611,15 @@ struct SPIRVEmitContext
                     emitOperand(ensureInst(sampledType));
                     break;
                 }
+                case kIROp_SPIRVAsmOperandImagePointer:
```

Comment:
As commented above, I don't think we need this case, since we can just do `&var`.

Reply:
commented why the workaround is required, an exact pattern of what happens if we do not will be attached in a comment here (with why it does not work)

Reply:
I think it will work if the image parameter is __ref.

Reply:
Image objects are not an L value, I don't believe __ref will work.